src/shared/OpenFiscaAPIClient.js

```diff
@@ -1,23 +1,9 @@
 import axios from "axios/index";
-import isDevelopment from "./isDevelopment";
+import { API_URL } from "../config"
 
 class OpenFiscaAPIClient {
-  url: ?string = undefined;
-  productionURL: string = "https://lesmevesajudes-api.herokuapp.com";
-  developmentURL: string = "http://localhost:2000";
-
-  constructor(isDevel: boolean) {
-    if (typeof isDevel === "undefined") {
-      isDevel = isDevelopment;
-    }
-    if (isDevel) {
-      this.url = this.developmentURL;
-    } else {
-      this.url = this.productionURL;
-    }
-  }
```

Comment:
Fail to see how removing this improves current code.

Reply:
The idea here was to use environment variables whenever possible, but it's just a suggestion :). There's some articles on why it's beneficial: https://serverless-stack.com/chapters/environments-in-create-react-app.html . But again, it was just an idea :)